Add Claude pre-review skill (#1142)
ai2cm/ace: .claude/skills/pre-review/SKILL.md

 - Catching and fixing any issues you can find before the PR is seen by the reviewer.
 - Making comments on the PR to point out potential issues or areas of concern for the reviewer to focus on.
 - Improving the PR description and title.

Focus specifically on making the reviewer's job easier, not on making the PR perfect.
Issues involving readability might be particularly important, especially if the changes are hard to understand or the reason for them is not immediately clear.

Begin by taking a look at the PR, as well as its description and title, and considering areas for improvement.
Communicate with the user about whether:
 - The PR has significant structural issues that should be fixed with significant refactors of the PR.
 - The PR should be significantly restructured, for example by restructuring the commits to have a more logical flow or by splitting into multiple smaller PRs.
 - The PR structure is acceptable, but there are specific issues that should be fixed before review.
 - The PR is in good shape, perhaps you have some comments to make on the PR or updates to its title/description but the code is ready for the reviewer.

> When leaving comments on a PR, always start your comment with "Claude: ".

Reviewer: It would be good IMO to keep this sort of thing harness-agnostic, e.g.
```suggestion
When leaving comments on a PR, always start your comment with "Pre-review agent: ".
```

Author: I'm a little wary of this cuing it to be less natural, but we can try it.

---
name: pre-review

> description: Prepare a PR for review by a team member. Use when the user wants to get a PR ready for review, make a PR easier to review, do a pre-review pass, or clean up a PR before requesting review. Also use when the user mentions preparing changes for a teammate to look at, or wants help making their branch review-ready.

Author: ```suggestion
description: Prepare a PR for review by a team member. Use when the user asks you to perform a pre-review.
```
